python/lsst/ts/planning/tool/zephyr_interface.py

```diff
@@ -133,14 +136,21 @@ async def get_list_of_statuses(self, status_type=None, max_results=20):
 
         return await self.get(endpoint, params)
 
-    async def get_steps_in_test_case(self, test_case_key):
+    async def get_steps(self, test_key, complement=True):
         """
-        Get all the steps in a test case.
+        Get all the steps in a test case or in a test execution.
+        The method will determine if the test key is a test case or a test
+        execution and will query the appropriate endpoint.
 
         Parameters
         ----------
-        test_case_key : str
-            The key of the test case.
+        test_key : str
+            The key of the test case or test execution.
+        parse : bool, optional
+            If True, the method will parse the payload. The default is True.
```

Comment:
You defining a `parse` parameter in the docstrings, but you are not using it. I think this is a missing thing as statuses of steps are not being parsed yet. I.e. I'm still getting, e.g.:
```
"status": {
    "id": 6360083,
    "self": "https://api.zephyrscale.smartbear.com/v2/statuses/6360083",
},
```